tests/Unit/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/Test_GaussianPlusConstant.cpp

```diff
@@ -19,15 +19,19 @@
 #include "Parallel/RegisterDerivedClassesWithCharm.hpp"
 #include "Utilities/Gsl.hpp"
 
-// \cond
+/// \cond
+namespace GeneralizedHarmonic {
+namespace ConstraintDamping {
 template <size_t VolumeDim, typename Fr>
 class GhDampingFunction;
```

Comment:
How about just `DampingFunction` instead of `GhDampingFunction` since it's already in the appropriate namespace?